third_party/mkl_dnn/mkldnn.BUILD

```diff
@@ -17,13 +17,24 @@ config_setting(
     },
 )
 
+template_rule(
+    name = "mkldnn_config_h",
+    src = "include/mkldnn_config.h.in",
+    out = "include/mkldnn_config.h",
+    substitutions = {
+        "#cmakedefine MKLDNN_CPU_BACKEND MKLDNN_BACKEND_${MKLDNN_CPU_BACKEND}": "#define MKLDNN_CPU_BACKEND MKLDNN_BACKEND_NATIVE",
+        "#cmakedefine MKLDNN_GPU_BACKEND MKLDNN_BACKEND_${MKLDNN_GPU_BACKEND}": "#define MKLDNN_GPU_BACKEND MKLDNN_BACKEND_NONE",
+    },
+)
+
 # Create the file mkldnn_version.h with MKL-DNN version numbers.
 # Currently, the version numbers are hard coded here. If MKL-DNN is upgraded then
 # the version numbers have to be updated manually. The version numbers can be
 # obtained from the PROJECT_VERSION settings in CMakeLists.txt. The variable is
 # set to "version_major.version_minor.version_patch". The git hash version can
 # be set to NA.
 # TODO(agramesh1) Automatically get the version numbers from CMakeLists.txt.
+# TODO(bhavanis): MKL-DNN version needs to be updated for MKL-DNN v1.x
```

Comment:
@penpornk Actually, it would work correctly if the current default MKL-DNN v0.18 is used. The only change needed for MKL-DNN v1.0 is that the minor version should be `95` instead of `18`.
Would it be okay if I change the minor version for v1.0 later?

Reply:
Got it. Could you please add this explanation as a comment in the file as well? (That right now it would still works even without the change.)